notebooks/langchain-api-client.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>This is pretty cool. If I understand it correctly, this class will have to be changed if we have a model mesh endpoint? In other words use a custom wrapper like the one here: https://python.langchain.com/en/latest/modules/models/llms/examples/custom_llm.html and then adapt it to our hosted api. How about using this file here: https://github.com/hwchase17/langchain/blob/master/langchain/llms/self_hosted_hugging_face.py</p><p>Seems like it would be very relevant to our use case?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
> https://github.com/hwchase17/langchain/blob/master/langchain/llms/self_hosted_hugging_face.py

The "self hosted hugging face" option does not work for us, it is meant for login-type access to remote hardware or virtual machines https://github.com/hwchase17/langchain/blob/aa11f7c89b2471e10a33b224387cb1f67b15b07b/langchain/llms/self_hosted_hugging_face.py#L112-L120

But yes, the other option (a custom LLM class) is what is implemented in this notebook :+1: